src/libraries/tests.proj

```diff
@@ -28,6 +28,11 @@
     <RunWasmSamples Condition="'$(RunSmokeTestsOnly)' != 'true' and '$(RunHighAOTResourceRequiringTestsOnly)' != 'true' and '$(TargetOS)' == 'browser' and '$(BuildAOTTestsOnHelix)' != 'true' and  '$(TestWasmBuildTests)' != 'true' and '$(TestWasmDebuggerTests)' != 'true'">true</RunWasmSamples>
   </PropertyGroup>
 
+  <!-- wasm EAT/AOT -->
+  <PropertyGroup Condition="'$(TargetOS)' == 'browser' and '$(RunDisabledWasmTests)' != 'true' and '$(EnableAggressiveTrimming)' == 'true'">
+    <JsonSerializerIsReflectionEnabledByDefault Condition="'$(RunAOTCompilation)' != 'true'">true</JsonSerializerIsReflectionEnabledByDefault>
```

Comment:
If testing depends on reflection serialization being available, I would recommend that this flag is turned on unconditionally so that the same components are run regardless of whether it's targeting AOT.

Reply:
Also, this should be moved to `eng/testing/tests.wasm.targets`

Reply:
> If testing depends on reflection serialization being available, I would recommend that this flag is turned on unconditionally so that the same components are run regardless of whether it's targeting AOT.

Yeah this was more me trying to figure out what why only that lane was failing, I had missed the disabling of the other TrimMode=full AOT lanes recently.